src/server/pfs/server/api_server.go

```diff
@@ -55,6 +56,18 @@ type apiServer struct {
 	_pachClient *client.APIClient
 }
 
+func compactPrintBranch(b *pfs.Branch) string {
+	return fmt.Sprintf("%s@%s", b.Repo.Name, b.Name)
+}
+
+func compactPrintCommit(c *pfs.Commit) string {
+	return fmt.Sprintf("%s@%s", c.Repo.Name, c.ID)
+}
+
+func compactPrintFile(f *pfs.File) string {
+	return fmt.Sprintf("%s@%s:%s", f.Commit.Repo.Name, f.Commit.ID, f.Path)
+}
```

Comment:
This would probably be better located in `src/server/pfs/pretty` I think there's some other similar functions there.